src/modules.py

```diff
@@ -132,7 +132,7 @@ def parse(config: Dict[str, Any], iocs: IocCollection) -> None:
 
     for name in ["email", "emails"]:
         for email in safe_get_list(config, name):
-            iocs.add_email(email)
+            iocs.add_email_to(email)
```

Comment:
Might cause some wrong results, but I guess it doesn't change the behaviour so the old was the same.